todo/src/repo/sequelize.ts

```diff
@@ -0,0 +1,67 @@
+const instances = require('hapi-sequelizejs').instances;
+
+export async function findAll() {
+    const [results] = await instances.dbs.mysql.sequelize.query
+    (`SELECT * FROM todo`);
+    return results
+}
+
+//GET a single todo
+export async function find(uuid) {
+    const [result] = await instances.dbs.mysql.sequelize.query
+    (`SELECT * FROM todo 
+    WHERE todo_uuid = '${uuid}'`);
+    return result;
+}
+
+// PUT update a todo
+export async function updateTodo(uuid, payload) {
+    var date = new Date(payload['dueDate']);
+    await instances.dbs.mysql.sequelize.query
+    (`UPDATE todo 
+    SET todo_todo = '${payload['todo']}',
+    todo_dueDate = '${date}',
+    todo_completed = ${payload['completed']}
+    WHERE todo_uuid = "${uuid}"`);
+    const [updatedTodo] = await instances.dbs.mysql.sequelize.query(`SELECT * FROM todo WHERE todo_uuid = "${uuid}"`);
+    return updatedTodo;
+}
+
+// POST create a new todo
+export async function create(newTodo) {
+    await instances.dbs.mysql.sequelize.query
+    (`INSERT INTO todo 
+    VALUES
+    ("${newTodo.uuid}", 
+    "${newTodo.todo}", 
+    "${newTodo.createdDate}",
+    "${newTodo.dueDate}",
+    ${newTodo.complete})`);
+    const [result] = await instances.dbs.mysql.sequelize.query('SELECT * FROM todo');
+    return await result
+}
+
+// DEL Delete from database
+export async function del(uuid) {
+    await instances.dbs.mysql.sequelize.query
+    (`DELETE FROM todo 
+    WHERE todo_uuid='${uuid}'`);
+    const [result] = await instances.dbs.mysql.sequelize.query('SELECT * FROM todo');
+    return result
+}
+
+// GET incompleted tasks
+export async function incompleteTodos() {
+    const[result] = await instances.dbs.mysql.sequelize.query
+    (`SELECT * FROM todo 
+    WHERE todo_completed = 0`);
+    return result;
+}
+
```

Comment:
Interpolating raw queries with user input like this makes you vulnerable to [SQL injectuon](https://www.w3schools.com/sql/sql_injection.asp)

Securing fully against SQL injection is outside the scope of the training but be aware that as a rule you should not ever allow user input directly into your queries due to SQL injection. You should instead [sanitize](https://www.quora.com/What-exactly-is-data-sanitization-with-respect-to-SQL-injection) the user input before it is used for a data query.